default/stringtables/en.txt

```diff
@@ -6238,7 +6242,13 @@ GAIA_SPECIAL
 Gaia
 
 GAIA_SPECIAL_DESC
-'''Terraforms planets then increases max Population by planet size; Tiny: +3, Small: +6, Medium: +9, Large: +12, Huge: +15 and increases max [[encyclopedia HAPPINESS_TITLE]] by 5.
+'''Terraforms planets then increases max Population by planet size:
```

Comment:
I used "bulleted list" style only when:
- there's more than one bonus (i.e. Population bonus and Happiness bonus)
- it's a Special, because the description is also displayed in a tooltip when you hover the Special icon in the Sidepanel (make the bonus value more obvious to see at first glance)